docs/src/content/docs/pt-pt/resources/plugins.mdx

```diff
@@ -0,0 +1,91 @@
+---
+title: Plugins e Integrações
+description: Descubra ferramentas da comunidade e integrações que estendem o Starlight!
+sidebar:
+  order: 1
+---
+
+:::tip[Inclua o seu!]
+Já construiu um plugin ou uma ferramenta para o Starlight?
+Então abra um PR e adicione o seu link nesta página!
+:::
+
+## Plugins
+
+Os [plugins](/pt-pt/reference/plugins/) podem personalizar ou estender a configuração do Starlight, a interface do utilizador e o comportamento, sendo também fáceis de compartilhar e de reutilizar.
+
+Estenda o seu site com os plugins oficiais suportados pela equipa Starlight e com plugins da comunidade mantidos por outros utilizadores Starlight.
+
+### Plugins oficiais
+
+<CardGrid>
+	<LinkCard
+		href="/pt-br/guides/site-search/#algolia-docsearch"
+		title="Algolia DocSearch"
+		description="Substitua o fornecedor de pesquisa Pagefind pelo Algolia DocSearch."
+	/>
+</CardGrid>
+
+### Plugins da comunidade
+
+<CardGrid>
+	<LinkCard
+		href="https://github.com/HiDeoo/starlight-links-validator"
+		title="starlight-links-validator"
+		description="Verifique links incorretos nas suas páginas do Starlight."
+	/>
+	<LinkCard
+		href="https://github.com/HiDeoo/starlight-typedoc"
+		title="starlight-typedoc"
+		description="Crie páginas Starlight a partir de TypeScript utilizando o TypeDoc."
+	/>
+	<LinkCard
+		href="https://github.com/HiDeoo/starlight-blog"
+		title="starlight-blog"
+		description="Adicione um blog ao seu site de documentação."
+	/>
+	<LinkCard
+		href="https://github.com/HiDeoo/starlight-openapi"
+		title="starlight-openapi"
+		description="Crie páginas de documentação a partir de especificações OpenAPI/Swagger."
+	/>
+	<LinkCard
+		href="https://github.com/HiDeoo/starlight-obsidian"
+		title="starlight-obsidian"
+		description="Publique cofres de Obsidian no seu site Starlight."
+	/>
+	<LinkCard
+		href="https://astro-ghostcms.xyz/intro/starlight/install/"
+		title="starlight-ghostcms"
+		description="Adicione os artigos do seu blog GhostCMS ao lado das páginas Starlight."
+	/>
+</CardGrid>
```

Comment:
Looks like one newer card for `starlight-image-zoom` needs adding here, see https://github.com/withastro/starlight/commit/8181a90d19b2748caf424e0c9b08fc3d40aec53f.